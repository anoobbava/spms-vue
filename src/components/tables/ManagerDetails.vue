<template>
  <h3> display manager details</h3>
</template>

<script>
export default {

}
</script>

<style>

</style>
