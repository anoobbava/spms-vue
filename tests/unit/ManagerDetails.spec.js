import { shallowMount, createLocalVue } from '@vue/test-utils'
import VueRouter from 'vue-router'
import ManagerDetails from '@/components/tables/ManagerDetails.vue'
import Vuex from 'vuex'

describe('ManagerDetails', () => {
  const localVue = createLocalVue()
  localVue.use(VueRouter)
  localVue.use(Vuex)
  const router = new VueRouter()
  const store = new Vuex.Store({
    state: {
      projects: { name: 'Project1' },
      projectNames: ['Project1']
    },
    getters: {
      projects: (state) => state.projects,
      projectNames: state => state.projectNames
    }
  })
  const wrapper = shallowMount(ManagerDetails, { localVue, router, store })

  it('is a component', () => {
    expect(wrapper.isVueInstance()).toBeTruthy()
  })

  it('renders correctly', () => {
    expect(wrapper.$el).toMatchSnapshot()
  })
})
