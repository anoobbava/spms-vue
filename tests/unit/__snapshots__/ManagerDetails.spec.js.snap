// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ManagerDetails renders correctly 1`] = `undefined`;
